skins/laika/src/pages/contact_form.ts

```diff
@@ -3,20 +3,21 @@ import VueI18n from 'vue-i18n';
 import PageDataInitializer from '@/page_data_initializer';
 import { DEFAULT_LOCALE } from '@/locales';
 import App from '@/components/App.vue';
-
 import Component from '@/components/pages/Contact.vue';
 import Sidebar from '@/components/layout/Sidebar.vue';
+import { ContactFormValidation } from '@/view_models/Validation';
 
 const PAGE_IDENTIFIER = 'contact-form';
 
 Vue.config.productionTip = false;
 Vue.use( VueI18n );
 
-interface ErrorModel {
-	message: string
+interface ContactFormModel {
+	message: string,
+	contactFormValidationPatterns: ContactFormValidation
 }
 
-const pageData = new PageDataInitializer<ErrorModel>( '#app' );
+const pageData = new PageDataInitializer<ContactFormModel>( '#app' );
```

Comment:
Good catch of a copy-paste error!

When we developed laika, we started out with the error page and took its page as the template for all other pages.